common/logger/src/struct_log.rs

```diff
@@ -1,76 +1,32 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
-use crate::{
-    counters::{STRUCT_LOG_CONNECT_ERROR_COUNT, STRUCT_LOG_TCP_CONNECT_COUNT},
-    Level,
-};
-use chrono::{SecondsFormat, Utc};
-use once_cell::sync::Lazy;
+use crate::counters::{STRUCT_LOG_CONNECT_ERROR_COUNT, STRUCT_LOG_TCP_CONNECT_COUNT};
 use serde::Serialize;
 use serde_json::Value;
 use std::{
     collections::HashMap,
-    env,
     fmt::Display,
     io,
     io::Write,
     marker::PhantomData,
     net::{TcpStream, ToSocketAddrs},
-    str::FromStr,
-    sync::atomic::{AtomicUsize, Ordering},
     time::Duration,
 };
 
-#[cfg(test)]
-mod tests;
-
-pub trait StructLogSink: Sync {
-    fn send(&self, entry: StructuredLogEntry);
-}
-
-// This is poor's man AtomicReference from crossbeam
-// It have few unsafe lines, but does not require extra dependency
-static NOP: NopStructLog = NopStructLog {};
-static mut STRUCT_LOGGER: &'static dyn StructLogSink = &NOP;
-static STRUCT_LOGGER_STATE: AtomicUsize = AtomicUsize::new(UNINITIALIZED);
-const UNINITIALIZED: usize = 0;
-const INITIALIZING: usize = 1;
-const INITIALIZED: usize = 2;
-
-// Size configurations
-const MAX_LOG_LINE_SIZE: usize = 10240; // 10KiB
-const LOG_INFO_OFFSET: usize = 256;
 const WRITE_TIMEOUT_MS: u64 = 2000;
 const CONNECTION_TIMEOUT_MS: u64 = 5000;
 
-// Fields to keep when over size
-static FIELDS_TO_KEEP: &[&str] = &["error"];
-
 #[derive(Debug, Serialize)]
 pub struct StructuredLogEntry {
```

Comment:
What's this used for at this point?

Reply:
I didn't want to go through the effort yet to remove all uses from the codebase. So right now its used everywhere, where people are creating structured logs without defining their own schemas